src/lib/y2network/interface_config_builders/br.rb

```diff
@@ -0,0 +1,71 @@
+require "yast"
+require "y2network/config"
+require "y2network/interface_config_builder"
+
+module Y2Network
+  module InterfaceConfigBuilders
+    class Br < InterfaceConfigBuilder
+
+      include Yast::Logger
+
+      def initialize
+        super(type: "br")
+      end
+
+      # @return [Array<Interface>] list of interfaces usable in bridge
+      def select_bridgeable
```

Comment:
the name is strange. You do not select anything. please use simple `bridgable` or `bridgable_interfaces`

Reply:
done for both br, bond